omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml

```diff
@@ -1,548 +1,815 @@
+				var $iycfField = jq(IYCF_FIELD).find('input');
+				var $arvsToMotherField = getField('arvs-to-mother.value');
+				var $gestationWeeksField = getField('gestation-weeks.value');
+				var $diagnosisParentFields = $('span.parent-diagnosis').find('input[type=checkbox]');
+				var $diagnosisChildFields = $('span.child-diagnoses').find('input[type=checkbox]');
+				var $encounterDateField = jq('#encounterDate').find('input[type="hidden"]');
+				var $encounterDate = jq('#encounterDate').find('input[type="hidden"]').val();
+				var $muacQuantitativeField = getField('muac-quantitative.value');
+				var $muacQualitativeField = getField('muac-qualitative.value');
+				var $emtctCodeWomanField = getField('emtct-code-woman.value');
+				var $preArtNumberField = jq('span#pre-art-number').find('input[type="text"]');
+		<!-- disable other field on Active Management of the 3rd stage of labour -->
+		jq('#99619').click(function () {
+                          if (getValue('165169.value') == 165125) {
+                            enableContainer('#1646');
+                          } else {
+                            disableContainer('#1646');
+                          }
+					  });
+		<!--  add or remove follow up visit    -->
+		var $allTables,
+						 $allRows,
+						 $otherRows,
+						 $firstRow,
+						 $thisRow,
+						 $nextRow,
+						 $hiddenRows,
+						 $lastVisibleRow,
+						 $nonEmptyBoxes;
+					 var showAddButtonForLastVisibleRow = function($container) {
+						 $lastVisibleRow = $container.find('tr:visible:last');
+						 $lastVisibleRow.find('.add').show();
+					 };
+
+					 jq('.FamilyTrackingEnableDisable').change(function(){ });
+
+
+					 $firstRow = jq('table.multi tbody tr:first-of-type');
+					 $otherRows = jq('table.multi tbody tr:not(:first-of-type)');
+
+					 jq('.add').click(function(e) { //Handle add button event
+						 e.preventDefault();
+						 $thisRow = jq(this).parents('tr:first');
+						 $nextRow = $thisRow.next();
+
+					 if ($nextRow.length &gt; 0) {
+						 $nextRow.show('slow');
+						 $thisRow.find('.add').hide();
+					 }
+
+					 showAddButtonForLastVisibleRow($thisRow.parents('table:first')); });
+
+					 jq('.remove').click(function(e) { //Handle remove button event
+						 e.preventDefault();
+						 $thisRow = jq(this).parents('tr:first');
+						 if (confirm('Are you sure you want to remove this record? All fields for this record will be cleared.') == 1) {
+							 $thisRow = jq(this).parents('tr:first');
+
+							 $thisRow.hide('slow', function() {
+							 showAddButtonForLastVisibleRow($thisRow.parents('table:first'));
+							 fieldHelper.clearAllFields($thisRow);
+						 });
+						 }
+					 });
+		<ifMode mode="ENTER">
+						 $otherRows.hide(); //Hide all rows apart from first row if adding new form
+						 $firstRow.find('.remove').hide(); //Hide the remove button from the first row
+
+						 jq('.FamilyTrackingEnableDisable').each(function(){
```

Comment:
@ODORA0 change this id from Family tracking to an appropriate ID

Reply:
I'll remove it